scripts/image-builder/config/blueprint_v0.0.1.toml

```diff
@@ -52,4 +52,4 @@ version = "*"
 [customizations]
 
 [customizations.services]
-enabled = ["crio", "microshift"]
+enabled = ["crio", "microshift", "microshift-ostree-pre-run"]
```

Comment:
Can we avoid enabling the service if it's an explicit pre-run dependency of `microshift`?

Reply:
From what I've seen during testing - probably no, I needed to enable it for "dependency links" to work